core/new-gui/custom-webpack.config.js

```diff
@@ -0,0 +1,11 @@
+module.exports = {
```

Comment:
I suggest we also add some comments here in the code, to link to the tutorial/example online about how to maintain this file, and why this file is needed.

Reply:
It is included in the PR description. Comments are not allowed in this file.